lib/postprocess/image.js

```diff
@@ -93,6 +95,19 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          sizeOf(imageBuffer)

```

Comment:
@sebastian-schlecht I don't know if this is the most elegant way to check for a valid buffer, I would look further to see if there are other possibilities.